RecoLocalTracker/SubCollectionProducers/src/TrackClusterRemover.cc

```diff
@@ -41,7 +43,7 @@ namespace {
 
     using QualityMaskCollection = std::vector<unsigned char>;
 
-    const float maxChi2_;
+    const unsigned char maxChi2_;
```

Comment:
similar comment to https://github.com/cms-sw/cmssw/pull/17098/files#r94253630
please add something to indicate that this is not a proper chi2